pyfiglet/__init__.py

```diff
@@ -906,6 +917,10 @@ def main():
     parser.add_option('-w', '--width', type='int', default=80, metavar='COLS',
                       help='set terminal width for wrapping/justification '
                            '(default: %default)')
+    parser.add_option('-b', '--blank', type='str', default=' ', metavar='BLANK',
+                      help='set blank character (default: "%default")')
+    parser.add_option('-e', '--exact', action='store_true', default=False,
+                      help='add additional blank characters to keep exact width')
```

Comment:
What do you think to calling `--exact` `--add-trailing-space` instead? `--exact` isn't very meaningful for me, but perhaps you and I can come up with something which would make it more obvious.

Reply:
Anything you want, I'm more interested in functionality than in syntax :)

Reply:
`-add-trailing-space` is obvious enough, anyway

Reply:
Sorry I didn't respond in a while. How about `--trailing-space` with short option `-T`? If you make the change, I'll be likely to accept. I'm also likely to accept other spellings if you have something better than `--exact`.

Reply:
Ahh, this was the point when I've lost the point ☹. So sorry for such a delay — didn't have resources to sort this out.

To be clear: you want me to rename _both_ of options, or just `--exact` one? For me `-b/--blank` looks descriptive enough — this is the character we will use __in place of blank__.

The meaning of `--excact` is «all lines of banner should be exact one width». I agree that «exact» alone tells nothing, but I believe it's important to keep direct «equal width» meaning, not just «add trailing characters» (also: they're not just spaces anymore☺). How about «`--equal-wdth`  / `-e`»?

But if you vote for `--trailing-space`/`-T` — it's OK, I'll accept that ☺

So what can I do to accelerate this?

Reply:
Apologies, I somehow missed this response before. As you can see, I come back to this project every once in a rare while. And apologies for holding it up with naming bike-shedding. Let's try and complete this.

Thinking about for a few minutes and coming back to it, I would accept:

* `--exact-width` ('exact' on its own could mean something else, adding the word 'width' is more descriptive)
* `--extra-padding`
* `--trailing-space`
* More suggestions.

I should have a small amount of time to action this in the coming Fridays, so feel free to ping the thread on a weekly basis if you don't hear back, and if I don't hear back, I'll make the adjustments at some point in the future and include the feature.